axon-framework/events/event-bus-and-event-store.md

```diff
@@ -98,7 +98,7 @@ Below is an example configuration of a persistence context configuration:
 
 By default, the `JpaEventStorageEngine` requires an `EntityManagerProvider` implementation that returns the `EntityManager` instance for the `EventStorageEngine` to use. This also allows application managed persistence contexts to be used. It is the `EntityManagerProvider`'s responsibility to provide a correct instance of the `EntityManager`.
 
-There are a few implementations of the `EntityManagerProvider` available, each for different needs. The `SimpleEntityManagerProvider` simply returns the `EntityManager` instance which is given to it at construction time. This makes the implementation a simple option for container managed contexts. Alternatively, there is the `ContainerManagedEntityManagerProvider`, which returns the default persistence context, and is used by default by the JPA event store.
+There are a few implementations of the `EntityManagerProvider` available, each for different needs. The `SimpleEntityManagerProvider` simply returns the `EntityManager` instance which is given to it at construction time. This makes the implementation a simple option for container managed contexts. Alternatively, there is the `ContainerManagedEntityManagerProvider`, which returns the default persistence context, and is used by default by the JPA event store. Please note these classes are duplicated to accommodate for the `javax` to `jakarta` namespace change.
```

Comment:
After seeing both changes, I think it's nicer to simply add a notice (similar to `> **Unique Key Constraint Consideration**`) about the `Jakarta or Javax?` question. That way, the pointers aren't spread in two spots, but combined in one. Furthermore, that means all documentation can focus on the Jakarta/current version instead, as this is where we want to steer the users to too.

Reply:
Not sure the update is what you had in mind, please check.